CHANGELOG.md

```diff
@@ -99,6 +99,7 @@
 * [#5173](https://github.com/bbatsov/rubocop/pull/5173): Add new `Style/EvalWithLocation` cop. ([@pocke][])
 * [#5208](https://github.com/bbatsov/rubocop/pull/5208): Add new `Rails/Presence` cop. ([@wata727][])
 * Allow auto-correction of ClassAndModuleChildren. ([@siggymcfried][], [@melch][])
+* [#3437](https://github.com/bbatsov/rubocop/issues/3437): Add new `Lint/NumberConversion` cop. ([@albertpaulp][])
```

Comment:
You should move this line to the section of the changelog for the current release.

Reply:
Moved cop info to latest release category.